docs/content/dagster-cloud/developing-testing/environment-variables-and-secrets.mdx

```diff
@@ -0,0 +1,335 @@
+          </li>
+        </ul>
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You want to manage environment variables in Dagster Cloud
+          </li>
+          <li>TODO: What else?</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/developing-testing/environment-variables/setup-for-serverless-deployments#configuring-environment-variables-in-github">
+          GitHub secrets
+        </a>
+      </td>
+      <td>
+        Environment variables are defined in a GitHub repository using Action
+        Secrets. <strong>Supported for Serverless deployments only</strong>.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            TODO: not sure if we're keeping this method
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/developing-testing/environment-variables/setup-for-serverless-deployments#passing-environment-variables-using-the-dagster-cloud-cli">
+          dagster-cloud CLI
+        </a>
+      </td>
+      <td>
+        Environment variables are passed in using the <code>--env</code> flag in
+        the CLI. <strong>Supported for Serverless deployments only</strong>.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You're using{" "}
+            <a href="/dagster-cloud/deployment/serverless">
+              Serverless deployment
+            </a>
+          </li>
+          <li>You're not using Dagster's GitHub app</li>
+          <li>
+            You want to manage environment variables outside of Dagster Cloud
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/developing-testing/environment-variables/setup-for-hybrid-deployments">
+          Agent configuration
+        </a>
+      </td>
+      <td>
+        Environment variables are defined in the agent's configuration in your
+        project's <code>dagster_cloud.yaml</code> file.{" "}
+        <strong>Supported for Hybrid deployments only</strong>.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You're using{" "}
+            <a href="/dagster-cloud/deployment/hybrid">Hybrid deployment</a>
+          </li>
+          <li>
+            You want to manage environment variables outside of Dagster Cloud
```

Comment:
This would cover things like setting kubernetes secrets right? I think this is for when you don't want Dagster Cloud to know what the values are (it all happens in your agent)

Reply:
Yeah; that's what I was thinking. I can re-work this to be more specific.